.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
```

Comment:
👍 great! Perhaps we should suggest people install [this](https://github.com/sindresorhus/atom-editorconfig) if they use atom in a contributing/readme somewhere so that this is enforced?